.circleci/config.yml

```diff
@@ -205,9 +216,19 @@ jobs:
           name: Deploy to gcloud test project
           command: |
             if [ "${CIRCLE_BRANCH}" == "master" ]; then
-              ./ci/deploy_ui.sh
+              ./project.rb deploy-ui \
+                --project all-of-us-workbench-test \
+                --account circle-deploy-account@all-of-us-workbench-test.iam.gserviceaccount.com \
+                --version circle-ci-test \
+                --promote
+            elif [ "${CIRCLE_TAG}" == "api_v*" ]; then
```

Comment:
Why aren't we checking for the api_v prefix anymore?

Reply:
I wasn't actually sure if we had a convention here. I imagined we'd want to push both the API and UI in one go, using a single tag. Therefore I wasn't sure we would be going with a convention of "api_v*". Might make more sense to switch to standard semantic versioning.

Thoughts on this? Did you have a convention in mind already?